Sources/PitchSpeller/Wetherfield/EdgeCapacity.swift

```diff
@@ -13,6 +13,7 @@ enum EdgeCapacity:
 }
 
 // MARK: - Comparable
+// bigM edges always 'win' comparisons - otherwise like a regular comparison
```

Comment:
I am liking this, though I wonder what the difference between this and using `Double.infinity` would be.

Reply:
None! Is that a thing ? (whoops!)

Reply:
👍